src/admin-views/admin-update-message.php

```diff
@@ -6,7 +6,11 @@
 
 ?>
 
-<p class="tribe-welcome-message"><?php echo esc_html( sprintf( __( 'You are running Version %s and deserve a hug :-)', 'the-events-calendar' ), Tribe__Events__Main::VERSION ) ); ?></p>
+
+<p class="tribe-update-message"><?php
```

Comment:
:no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 9456cc28da099eed3f9c7c00d8d4c804</sub>